pkg/manager/manager.go

```diff
@@ -1228,7 +1228,8 @@ func (m *jobManager) ResolveAsPullRequest(spec string) (*prowapiv1.Refs, error)
 	if len(parts) != 2 {
 		return nil, nil
 	}
-	locationParts := strings.Split(parts[0], "/")
+	location := strings.TrimSpace(parts[0])
+	locationParts := strings.Split(location, "/")
```

Comment:
Hey @stephenfin,
I'd recommend that this fix, or something similar, be moved to a different location in the code...
That way it would benefit all the commands instead of just the `build` command you're attempting to run.
I'll let you decide if you want me to tell you where that is or if you'd like to find it yourself.

Reply:
Good point. Done.